Himotoki/decode.swift

```diff
@@ -22,6 +27,11 @@ public func decodeArray<T: Decodable where T.DecodedType == T>(object: AnyObject
 }
 
 /// - Throws: DecodeError
+public func decodeArray<T: Decodable where T.DecodedType == T>(object: AnyObject, rootKeyPath: KeyPath) throws -> [T] {
```

Comment:
Length Violation: Line should be 100 characters or less: currently 119 characters